tests/helpers/__init__.py

```diff
@@ -41,15 +48,109 @@ def create_random_packet(root, name="data", *, parameters=None, packet_id=None):
     return p.id
 
 
+## Create a chain of packets a, b, c, ... that depend on each other
+def create_random_packet_chain(root, length, base=None):
+    ids = {}
+    with TemporaryDirectory() as src:
+        for i in range(length):
+            name = chr(i + ord("a"))
+            packet_id = outpack_id()
+            ids[name] = packet_id
+            packet_path = Path(src) / name / packet_id
+            os.makedirs(packet_path)
+
+            packet = Packet(root, packet_path, name, id=packet_id, locate=False)
+
+            if i == 0 and base is None:
+                with open(packet_path / "data.txt", "w") as f:
+                    f.write("0")
+            else:
+                lines = [
+                    "with open('input.txt', 'r') as f:",
+                    "    data = f.read()",
+                    "with open('data.txt', 'w') as f:",
+                    f"    f.write(data + '{i}')",
+                ]
```

Comment:
Writing a script and evaluating it seems a bit overkill. Why don't we just write the data directly from this function?